platform/mellanox/mlnx-issu/scripts/ffb

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+"""
+Part of MLNX platform specific fast-fast boot implementation for warm-boot.
+Notifies SYNCD proccess once boot is finished after warm-reboot.
+Once SYNCD received such notification it should set appropriate SAI attribute.
+Then SAI will notify SDK to end ISSU mode for the FFB.
+"""
+
+
+import time
+import swsssdk
```

Comment:
use swsscommon python bind, it has notificationconsumer implemented, do not use swssdk.

Reply:
```swsssdk``` is used here because ```swsscommon``` is not available inside ```syncd``` docker container, only ```swsssdk```